src/ui/mod.rs

```diff
@@ -239,7 +239,11 @@ impl Ui {
                         ui.set_width(250.0);
 
                         if ui.button(fl!("menu_language_system")).clicked() {
-                            select_system_locales();
+                            // Not sure if unwrap is the best idea here, but I don't know enough
+                            // about the app to say other wise
+                            //
+                            // Feel free to replace
+                            select_system_locales().unwrap();
```

Comment:
Recommand not to unwrap here. You can just ignore it or simply log the error (better).